api/overdrive.py

```diff
@@ -836,7 +836,7 @@ def release_hold(self, patron, pin, licensepool):
         url = self.HOLD_ENDPOINT % dict(
             product_id=licensepool.identifier.identifier)
         response = self.patron_request(patron, pin, url, method='DELETE')
-        if response.status_code / 100 == 2 or response.status_code == 404:
+        if response.status_code // 100 == 2 or response.status_code == 404:
```

Comment:
Unrelated change for Python 3 forward-compatibility, found while grepping for "100".